### PR TITLE
Added requirements

### PR DESCRIPTION
**Pull request by kingspp**

### CODE REVIEW

**Comment by shz1314** [comment]
Hi, it is an outstanding work! Very thanks. I want to follow your work in how to optimize the loss. Have you released all training code? Shall you give me some advice？

**Comment by julycetc** [comment]
Hello,do you have search and train code for cifar?Thanks very much!
